### PR TITLE
Use angle-bracket imports when importing headers from other modules

### PR DESCRIPTION
**Pull request by segiddins**
cc @justinseanmartin

### CODE REVIEW

File: Aardvark/ARKEmailBugReporter.m [hunk 1]
```diff
@@ -21,12 +21,13 @@
 #import "ARKEmailBugReporter.h"
 #import "ARKEmailBugReporter_Testing.h"
 
-#import "AardvarkDefines.h"
 #import "ARKDefaultLogFormatter.h"
 #import "ARKScreenshotLogging.h"
-#import "ARKLogStore.h"
 #import "ARKLogMessage.h"
 
+#import <CoreAardvark/AardvarkDefines.h>
```
Comment (dfed) [inline Aardvark/ARKEmailBugReporter.m]: Good find! We should put imports from other modules above local imports though, right?

File: Aardvark/ARKEmailBugReporter.m [hunk 2]
```diff
@@ -21,12 +21,13 @@
 #import "ARKEmailBugReporter.h"
 #import "ARKEmailBugReporter_Testing.h"
 
-#import "AardvarkDefines.h"
 #import "ARKDefaultLogFormatter.h"
 #import "ARKScreenshotLogging.h"
-#import "ARKLogStore.h"
 #import "ARKLogMessage.h"
```
Comment (dfed) [inline Aardvark/ARKEmailBugReporter.m]: This is `CoreAardvark` as well I believe

**Comment by justinseanmartin** [comment]
Want to include a bugfix version bump with this PR?

**Comment by segiddins** [comment]
Changes made

**Comment by dfed** [comment]
Thank you folks!!

**Comment by justinseanmartin** [comment]
2.0.2 is now published to CocoaPods trunk.

**Pull request merged**